.jenkins/Jenkinsfile

```diff
@@ -14,8 +14,8 @@ def ACCTest(String label, String compiler, String build_type) {
                     }
                     withEnv(["CC=${c_compiler}","CXX=${cpp_compiler}"]) {
                         sh """
-                        cmake ${WORKSPACE} -DCMAKE_BUILD_TYPE=${build_type}
-                        make
+                        cmake ${WORKSPACE} -G Ninja -DCMAKE_BUILD_TYPE=${build_type}
+                        ninja
```

Comment:
This is great! I would suggest `ninja -v` since these are CI runs, so more verbosity is probably better (and Ninja's default output is very quiet).

Reply:
Yes that sounds good to me too. I like more verbosity in CI usually! :)

Reply:
Can you also do a bors try on this PR so that we can see the extra output with -v?

Reply:
Good idea! Done.